brave/browser-laptop
# Remove Base64 encoded images from `preferences.less`

## Test Plan
- Open up content dev tools using F12 or alt + option + i
- Select the two images shown below in preferences > payments
- ensure logo is a PNG file

## Details
Fixes #5224


![screen shot 2017-02-13 at 10 36 29](https://cloud.githubusercontent.com/assets/347657/22878054/79244d80-f1d8-11e6-95fe-8de26a09c448.png)
![screen shot 2017-02-13 at 10 36 21](https://cloud.githubusercontent.com/assets/347657/22878055/7925234a-f1d8-11e6-9bf0-45810fbe67f4.png)
